rust/otap-dataflow/crates/otap/Cargo.toml

```diff
@@ -63,13 +63,29 @@ zip.workspace = true
 
 # Geneva exporter dependencies
 geneva-uploader = { version = "0.3.0", optional = true }
-opentelemetry-proto = { workspace = true, optional = true }
+
+# GigLA exporter dependencies
+azure_identity = { version = "0.29.0", optional = true }
+flate2 = { version = "1.1.5", optional = true }
+reqwest = { version = "0.12.24", optional = true }
+azure_core = { version = "0.29.1", optional = true }
```

Comment:
As a good practice for repo maintenance, we have defined the crate versions in the higher level [`Cargo.toml`](https://github.com/open-telemetry/otel-arrow/blob/main/rust/otap-dataflow/Cargo.toml#L48-L135) where we have defined the workspace. Please define the crate version there and use the `.workspace = true` approach of using the dependency here.

Reply:
Yes, please. If Rust has anything like the gomod hell I've seen, it comes from mixing dependencies across crates in a workspace! (and/or I haven't seen the worst of it yet) :-)